flocker/node/agents/test/test_blockdevice_manager.py

```diff
@@ -174,6 +174,7 @@ def test_bind_mount(self):
         src_directory = self._get_directory_for_mount()
         target_directory = self._get_directory_for_mount()
         self.manager_under_test.bind_mount(src_directory, target_directory)
+        self.addCleanup(self.manager_under_test.unmount, target_directory)
```

Comment:
:question: No action required here, but I wonder if we could have an API that avoids the need to remember to `addCleanup`